_src/assets/js/collapsible.js

```diff
@@ -9,38 +9,42 @@ const arrowElDesign = document.getElementById("dropdown__down-arrow--design");
 const arrowElFill = document.getElementById("dropdown__down-arrow--fill");
 const arrowElShare = document.getElementById("dropdown__down-arrow--share");
 
-const panelDesignEl = document.querySelector(".panel-design__radios");
+// TO BE REMOVED -- Apparently these 3 could be removed. Need approval to do so.
+const panelDesignEl = document.querySelector(".panel__design-radios");
 const panelFillEl = document.querySelector(".panel-fill__form");
 const panelShareEl = document.querySelector(".align_share");
 
+// Consts for listeners
+const designDropdownEl = document.querySelector(".panel__dropdown--design");
+const fillDropdownEl = document.querySelector(".panel__dropdown--fill");
+const shareDropdownEl = document.querySelector(".panel__dropdown--share");
+
```

Comment:
I think `panelDesignEl`, `panelFillEl` and `panelShareEl` could be made with a `querySelectorAll('panel__dropdown')` and go through them with a `for... of`. And this way we save ourselves from creating more constants.

Reply:
On it!

Reply:
Should be fixed!

Reply:
Perfect. You can merge